core/src/main/scala/io/finch/Error.scala

```diff
@@ -1,28 +1,44 @@
 package io.finch
 
+import scala.compat.Platform.EOL
 import scala.reflect.ClassTag
 
+import cats.data.NonEmptyList
+import io.finch.items.RequestItem
+
 /**
  * A basic error from a Finch application.
  */
-trait Error extends Exception
+trait Error extends Exception {
+  def canEqual(other: Any): Boolean = other.isInstanceOf[Error]
 
-object Error {
+  override def hashCode(): Int = (super.hashCode, getMessage).##
+
+  override def equals(other: Any): Boolean = other match {
+    case that: Error => that.canEqual(this) && that.getMessage == this.getMessage
+    case _ => false
+  }
+}
 
-  import items._
+object Error {
 
   def apply(message: String): Error = new Error {
     override def getMessage: String = message
   }
 
+  object RequestErrors {
+    @deprecated("Use Multiple instead", "0.11")
+    def unapply(e: Throwable): Option[Seq[Throwable]] = Some(Seq(e))
```

Comment:
As far as I understand, if someone is using a pattern match on `RequestErrors(_)` it returns the `Seq[Throwable]` that was wrapped in it (and it will show a deprecated warning) which will still be the type of Throwable right? otherwise the signature changes -> no backward compatibility. 

Should something like this suffice then?
```scala
e match {
      case Multiple(err) => Some(err.toList)
      case t: Throwable => Some(Seq(t))
      case _ => None
    }
```

or am I misunderstanding your "should be backed by Multiple(nel)"?